File: lib/data/notification.dart
import 'package:flutter_local_notifications/flutter_local_notifications.dart';
import 'package:timezone/timezone.dart';

class Notification {
  int id;
  String title;
  String body;
  TZDateTime schedule;
  DateTimeComponents matchDateTimeComponents;

  Notification({this.id, this.title, this.body, this.schedule, this.matchDateTimeComponents});
}

File: functions/notification.js
/* eslint-disable */

/**
  * This is the notification class
  */
class Notification {
  constructor(id, title, body, schedule) {
    this.id = id;
    this.title = title;
    this.body = body;
    this.schedule = schedule;
  }
}

const notificationData = [
  new Notification("00", "dentist", "you should go to the dentist soon...", "* * * * *"),
  new Notification("01", "water", "remember to drink enough water", "* * * * *"),
];

module.exports = { notificationData };